samples/net/wifi/shell/prj.conf

```diff
@@ -36,3 +36,5 @@ CONFIG_NET_L2_WIFI_SHELL=y
 # environment.
 CONFIG_NET_MGMT_EVENT_QUEUE_TIMEOUT=5000
 CONFIG_NET_MGMT_EVENT_QUEUE_SIZE=16
+
+CONFIG_NRF70_DEBUG=y
```

Comment:
This should be put to an overlay file so that is only enabled if needed.

Reply:
Sorry this was an accidental commit.